src/test/util/setup_common.cpp

```diff
@@ -180,8 +180,8 @@ ChainTestingSetup::ChainTestingSetup(const std::string& chainName, const std::ve
 
     const ChainstateManager::Options chainman_opts{
         .chainparams = chainparams,
+        .datadir = m_args.GetDataDirNet(),
         .adjusted_time_callback = GetAdjustedTime,
-        .check_block_index = true,
```

Comment:
Not going against this change, but this line removal doesn't look to be part of a "non-behavior change" commit.

Reply:
re: https://github.com/bitcoin/bitcoin/pull/25862#discussion_r1087877059

In commit "refactor, validation: Add ChainstateManagerOpts db options" (cde3882cd4d4d1ee5682cab47d05e786cc798eb8)

> Not going against this change, but this line removal doesn't look to be part of a "non-behavior change" commit.

Good catch. It looks like this line got dropped due to a bad rebase. It was not supposed to be deleted.